src/__test__/ios.test.ts

```diff
@@ -27,8 +27,8 @@ describe('test parse ios logs', () => {
 
         expect(id).toEqual('N/A');
         expect(start).toBeNull();
-        expect(end).toBeNull();
-        expect(elapsed).toEqual(0);
+        expect(end).toEqual(new Date('2018-06-04T22:31:40.000Z'));
+        expect(elapsed).toEqual(160.313);
```

Comment:
This is an unexpected change - what happened to turn end from null to an actual date?

Reply:
It was previously `null` because the log had an unsupported and unused datetime timezone int (GMT instead of UTC). I corrected the data to the actual timezone used in our logs.
This isn't an unexpected change in so much as it was bad test data.